lib/ex_aws/dynamo.ex

```diff
@@ -94,7 +94,8 @@ defmodule ExAws.Dynamo do
   ######################
 
   @type table_name :: binary
-  @type primary_key :: [{atom, binary}] | %{atom => binary}
+  @type primary_key_val :: number
```

Comment:
Should that not be `@type primary_key_val :: number | binary`?

Reply:
Thank you.